tensorstore/kvstore/tsgrpc/kvstore_server.cc

```diff
@@ -440,32 +507,37 @@ TENSORSTORE_DEFINE_JSON_DEFAULT_BINDER(
                }),
                jb::Member("bind_addresses",
                           jb::Projection<&KvStoreServer::Spec::bind_addresses>(
-                              jb::DefaultInitializedValue()))));
+                              jb::DefaultInitializedValue())),
+               jb::Member("max_sent_part_bytes",
```

Comment:
Hm, if we want to support a parameter like this, then it needs to be validated to be within the range [1, 4mb - ~16), otherwise it will cause gRPC issues.

Reply:
I actually considered such naming but decided that it would be sort of misleading, as this parameter limits the size of the "value_part" field of the protobuf and not the total size of the message (which is typically slightly larger due to the rest of the fields).

Anyway, this should not be a big deal, so let me change the naming as you propose.